src/ClassicUO.Client/LegionScripting/API.py

```
@@ -24,293 +25,1385 @@ class Notoriety:
+    """
+     Target an item or mobile.
+     Example:
+     ```py
+     if API.WaitForTarget():
+       API.Target(0x12345678)
+     ```
+    
+    """
     pass
 
 def Target(x: int, y: int, z: int, graphic: int = 1337):
```

Ruff (0.11.9)

Redefinition of unused `Target` from line 668

(F811)